ci-operator/step-registry/openstack/provision/sriov-networknodepolicy/openstack-provision-sriov-networknodepolicy-commands.sh

```diff
@@ -6,26 +6,26 @@ set -o pipefail
 
 wait_for_sriov_pods() {
     # Wait up to 15 minutes for SNO to be installed
-    for _ in $(seq 1 90); do
+    for _ in $(seq 1 15); do
         SNO_REPLICAS=$(oc get Deployment/sriov-network-operator -n openshift-sriov-network-operator -o jsonpath='{.status.readyReplicas}' || true)
         if [ "${SNO_REPLICAS}" == "1" ]; then
             FOUND_SNO=1
             break
         fi
         echo "Waiting for sriov-network-operator to be installed"
-        sleep 10
+        sleep 60
```

Comment:
this will make the step a bit more slower, because you wait more between retries. Why did you reduce the # of retries and increased the waiting time? I'm trying to understand this change.

Reply:
To give it time for also the other pods (daemon, webhook, injector) to at least be created, not necessarily Running. It might happen that if we get to the next check in line 23 and those pods weren't created yet, it will just pass as if everything is running.